internal/data/spicedb.go

```diff
@@ -21,19 +22,49 @@ type SpiceDbRepository struct {
 
 // NewSpiceDbRepository .
 func NewSpiceDbRepository(c *conf.Data, logger log.Logger) (*SpiceDbRepository, func(), error) {
+	return newSpiceDbRepository(c, "", "", "", logger)
+}
+
+func newSpiceDbRepository(c *conf.Data, tokenOverride string, endpointOverride string, useTlsOverride string, logger log.Logger) (*SpiceDbRepository, func(), error) {
```

Comment:
Just thinking about it, we *could* remove all the overrides by injecting a different `*conf.Data` for tests. But I'm not sure how to do this just yet, so probably future work.

Reply:
Fair, though I left a comment in the SpiceDB container- it knows/controls the relevant config values and is what calls this in tests; it seems like it could handle this